tools/xmlsec1/tests/fixtures/upstream/testKeys.sh

```
@@ -0,0 +1,613 @@
+        res=$?
+        if [ $res -ne 0 ]; then
+	        cat $curlogfile >> $logfile
+	        tearDownTest
+            return
+        fi
+    fi
+
+    # run tests
+
+    # generate key
+    if [ -n "$alg_name" -a -n "$key_name" -a "z$xmlsec_feature_gen_key" = "zyes" ]; then
```

Shellcheck (0.11.0)

[warning] Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

(SC2166)